src/localization/locales/es-ES.json

```diff
@@ -20,6 +20,8 @@
         "img": {
             "src": "groups.png"
         },
+        "aria_label": "groups-list-title",
+        "aria_label_default": "groups-default-title",
```

Comment:
question: this looks like a DOM ID but behaves like a localized string. What is it supposed to do?

Reply:
it labels the accessibility role for unit testing; since it's not dynamic, I just added it to the localization json files

Reply:
is there a better way to do this?

Reply:
is there a better way to do this?